code/__DEFINES/dcs/signals/signals_hydrophonic.dm

```diff
@@ -0,0 +1,35 @@
+//Plants / Plant Traits
```

Comment:
`signals_hydrophonic.dm` -> `signals_hydroponics.dm`, file name is misspelled

Reply:
ops thanks